proc.go

```diff
@@ -70,6 +70,8 @@ type Proc struct {
 		mu   sync.RWMutex
 		run  bool
 		loop bool
+
+		frameCount int64
```

Comment:
```suggestion
		nframes uint64
```

also, as it's a field protected by the `mu sync.RWMutex`, please make it part of the same block than `run` and `loop`.